### ISSUE TITLE
Build System: parallel install of different app versions

### ISSUE THREAD

**Issue by cascheberg** [mozilla/MozStumbler#1201]
I'd like to keep the Play Store version installed in any case, so it would be great if Github and local builds could be installed in parallel.
Whenever I built the app locally, I have to remove the Play Store version and all its data before I can run it on my device. Is there a way around that? Am I doing it wrong?
And it's nice to have one permanent stable version and one dev/debug/reference version installed.
![problem](https://cloud.githubusercontent.com/assets/3845150/4900772/8574edd4-642b-11e4-876e-f670fa7d4288.png)

**Comment by garvankeeley** [comment]
I think the idea of having both for comparison as we try out new features is a good one. 
We were planning to build the github and play store with the same signing key so that users could switch between them (hopefully seamlessly, it _should_ not require uninstalling). I don't think there is a possibility of having 2 builds with the same org.mozilla.mozstumbler package name. So if we aren't going to use the same signing key for the builds, than we should have 2 package names so that the two builds don't conflict. 

However, there is also the use case of attracting more users from the play store to the github build, to grow the 'beta' program. These users are more likely to want to install the github build over top of their play store build (with no uninstall), and then still have play store notify them of the next update (and switch back to play store without uninstalling).

**Comment by cascheberg** [comment]
In my focus of interest is the ability to install local builds while keeping one permanent official release.
I just found out that this can be done by adding 

> packageName "org.mozilla.mozstumbler_debug"

to one of the `productFlavors` in the gradle build file.
